src/components/Heading/HeadingHooks.ts

```diff
@@ -0,0 +1,22 @@
+import { HeadingSizes, HeadingTypes } from "./HeadingConstants";
+
+export function useLegacyHeadingClassNameByType(
```

Comment:
that's not exactly a hook but rather a simple function, no?

Reply:
i'm a huge fan of custom hook for readability but you right... it  will be better to set this as function, will fixed